src/statusBarItem.ts

```diff
@@ -8,8 +8,8 @@ export interface Clickable {
 export interface Status {
     type: 'ok' | 'error'
     title: string
-    quiescent: boolean,
-    clickable: Clickable | null
+    quiescent?: boolean,
```

Comment:
I think this would be clearer if `quiscient` is explicitly false or true.

My bad that the logic for `quiscient` is faulty, it should be the opposite, I'll send a PR to fix it after this is merged.

Reply:
Will let you handle that in the future